app/frontend/src/features/donations/DonationsBox.tsx

```diff
@@ -0,0 +1,203 @@
+    backgroundColor: theme.palette.primary.main,
+    "&&": {
+      borderRadius: "0.5rem",
+      boxShadow: "initial",
+    },
+    "&:hover": {
+      opacity: "0.4",
+      backgroundColor: theme.palette.primary.main,
+    },
+    "& .MuiButton-label": {
+      color: theme.palette.background.paper,
+      transition: "color 250ms cubic-bezier(0.4, 0, 0.2, 1)",
+    },
+  },
+
+  buttonMainText: {
+    color: theme.palette.background.paper,
+    fontWeight: 700,
+    transition: "color 250ms cubic-bezier(0.4, 0, 0.2, 1)",
+  },
+
+  marginY2: {
+    margin: theme.spacing(2, 0),
+  },
+
+  marginBottom2: {
+    marginBottom: theme.spacing(2),
+  },
+
+  inputWrapper: {
+    position: "relative",
+    "&::before": {
+      content: `'${DONATIONSBOX_CURRENCY}'`,
+      position: "absolute",
+      top: "50%",
+      transform: "translateY(-50%)",
+      left: theme.spacing(1),
+      color: theme.palette.grey[600],
+      fontWeight: 700,
+      fontSize: theme.typography.body1.fontSize,
+    },
+  },
+
+  inputNumber: {
+    width: "100%",
+    height: "100%",
+    borderRadius: theme.shape.borderRadius,
+    border: `2px solid ${theme.palette.grey[200]}`,
+    paddingLeft: theme.spacing(2),
+    color: theme.palette.grey[600],
+    fontWeight: 700,
+    fontSize: theme.typography.body1.fontSize,
+    "&&": {
+      borderRadius: "0.5rem",
+      boxShadow: "initial",
+    },
+    "&:hover": {
+      border: `2px solid ${theme.palette.primary.main}`,
+    },
+    "&:focus-visible": {
+      border: `2px solid ${theme.palette.primary.main}`,
+      outline: "none",
+      boxShadow: "none",
+    },
+  },
+}));
+
+export default function Donations() {
+  const classes = useStyles();
+
+  return (
+    <section className={classes.donationsBox}>
+      <Typography className={classes.marginBottom2} variant="h3">
+        {DONATIONSBOX_TITLE}
+      </Typography>
+      <div className={classes.donationsBoxRow}>
+        <Button
+          classes={{
+            root: classes.buttonSecondary,
+            label: classes.buttonSecondaryText,
+          }}
+        >
+          {DONATIONSBOX_MONTHLY}
+        </Button>
+        <Button
+          classes={{
+            root: classes.buttonSecondaryActive,
+            label: classes.buttonSecondaryTextActive,
+          }}
+        >
+          {DONATIONSBOX_ONETIME}
+        </Button>
+      </div>
+
+      <Divider className={classes.marginY2} />
+
+      <div className={classes.donationsBoxRow}>
+        {Array.from(DONATIONSBOX_VALUES?.values() ?? []).map((value) =>
+          value ? (
+            <Button
+              classes={{
+                root: classes.buttonSecondary,
+                label: classes.buttonSecondaryText,
+              }}
+            >
+              {`${value.currency}${value.amount}`}
+            </Button>
+          ) : null
+        )}
+        <div className={classes.inputWrapper}>
+          <input type="number" className={classes.inputNumber} />
```

Comment:
Use our TextField component, which by default is `variant="outlined"` and you may not need any custom styling (or much less):
```
<TextField
  type="number"
  InputProps={{startAdornment: <InputAdornment position="start">{DONATIONSBOX_CURRENCY}</InputAdornment>}}
/>

Reply:
I'm checking that out and it seems I have to overwrite lots of stuff there. Is it OK, if I leave the syled input?

Reply:
Yeah ok, I guess this one is very different

Reply:
In that case add an aria-label